### PR TITLE
MRG,DOC: convert references to bibtex

### PR DESCRIPTION
**Pull request by drammock**
WIP, chipping away at bibtex updates.

### CODE REVIEW

File: mne/io/meas_info.py [hunk 1]
```diff
@@ -67,16 +67,15 @@
...
-    therein.
+    The International System of Units (SI, :footcite:`WikipediaSI`) is the
+    default system for describing units in the Brain Imaging Data Structure
+    (BIDS). For more information, see the BIDS specification
+    :footcite:`BIDSdocs` and the appendix "Units" therein.
 
     References
     ----------
-    [1] .. https://en.wikipedia.org/wiki/International_System_of_Units
-    [2] .. https://bids-specification.readthedocs.io/en/stable/
```
Comment (larsoner) [inline mne/io/meas_info.py:77]: Azure complains about the newline
```suggestion
```

**Review by larsoner — COMMENTED** [review]
Otherwise LGTM

**Comment by drammock** [comment]
In the interest of smallish diffs, I'm keeping these BibTeX PRs to a manageable number of files at a time / saving them for when I'm waiting for a long-running job to complete :)

This one is ready for review/merge; Style and Circle are happy and I checked all the examples that were touched and they render correctly.

**Comment by larsoner** [comment]
Thanks @drammock

**Pull request merged**